CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1067,9 +1067,10 @@ private void SetPopupSizeAndPositionImpl(int width, int height, int x, int y)
             popup.Width = width;
```

Comment:
I think we need to scale the width and height from the results I'm seeing on Win7 at 1.25